docs/developer-docs/getting-started/quickstart/first-smart-contract.mdx

```diff
@@ -0,0 +1,168 @@
+---
+keywords: [beginner, getting started, first smart contract, first deployment, deploy a smart contract, quickstart, quick start]
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
+import TabItem from "@theme/TabItem";
+import { AdornedTabs } from "/src/components/Tabs/AdornedTabs";
+
+# Quick start
+
+<MarkdownChipRow labels={["Beginner", "Getting started" ]} />
+
+Build Web3, DeFi, fully network-resident social media, games, enterprise apps, and just about anything else you can imagine. Your projects are immune to cyber attacks and unstoppable.
+
+**Deploy in minutes using these steps.**
+
+<AdornedTabs>
+<TabItem value="linux" label="Linux" default={true}>
+
+#### Prerequisites
+
+- [x] Download and [install Node.js](https://nodejs.org/en/download/package-manager).
+
+#### 1. Install the IC SDK.
+
+```bash
+sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+source "$HOME/.local/share/dfx/env"
+```
+
+Confirm the IC SDK has been installed:
+
+```bash
+dfx --version
+```
+
+#### 2. Start `dfx`.
+
+```bash
+dfx start --background
+```
+
+#### 3. Create a new project.
+
+The default project created by `dfx` is "Hello, world!"
+
+
+```bash
+dfx new hello --type=motoko // Creates a new Motoko project
```

Comment:
I would suggest simplifying a bit by providing the additional options in the comment and removing the line below

```suggestion
dfx new hello --type=motoko // Creates a new Motoko project. Options are [motoko, rust, azle, kybra]
```